home/src/app/services/login/login.service.js

```diff
@@ -86,7 +86,11 @@
             sessionStorage.removeItem('userProfile');            
             sessionStorage.removeItem('accessToken');            
             $http.defaults.headers.common['Authorization'] = undefined;
-            delete $rootScope.userProfile;       
+            delete $rootScope.userProfile;
```

Comment:
It means when you using window.location.reload, you reload the entire page. This still works but not necessary. You can use $location.path to reload just a part of the page, which you want to change